files/zh-tw/web/http/status/425/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 425 Too Early
+slug: Web/HTTP/Status/425
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}}
+
+超文字傳輸協定（HTTP）**`425 Too Early`** 回應狀態碼表示伺服器不願意冒險處理可能被重送的請求，這會造成重送攻擊的潛在風險。
```

Comment:
- https://zh.wikipedia.org/zh-tw/%E8%B6%85%E6%96%87%E6%9C%AC%E4%BC%A0%E8%BE%93%E5%8D%8F%E8%AE%AE
- https://zh.wikipedia.org/zh-tw/%E9%87%8D%E6%94%BE%E6%94%BB%E5%87%BB

```suggestion
超文本傳輸協定（HTTP）**`425 Too Early`** 回應狀態碼表示伺服器不願意冒險處理可能被重送的請求，這會造成重放攻擊的潛在風險。
```